__test__/unit/helpers/mainHelper.test.ts

```diff
@@ -0,0 +1,26 @@
+import MainHelper from "../../../src/shared/helpers/MainHelper";
+import { SubscriptionStateKind } from "../../../src/shared/models/SubscriptionStateKind";
+import { HttpHttpsEnvironment } from "../../support/models/HttpHttpsEnvironment";
+import { TestEnvironment } from "../../support/environment/TestEnvironment";
+import { NotificationPermission } from "../../../src/shared/models/NotificationPermission";
+
+describe('MainHelper Tests', () => {
+  beforeEach(() => {
+    jest.useFakeTimers();
+    TestEnvironment.initialize({
```

Comment:
@rgomezp @jennantilla This test was creating some errors so I fixed it in this PR:
https://github.com/OneSignal/OneSignal-Website-SDK/pull/1056